docs/siem/detections/api/exceptions/api-create-exception-item.asciidoc

```diff
@@ -0,0 +1,372 @@
+|Name |Type |Description |Required
+
+|`comments` |comments[] a|Array of `comment` fields:
+
+* `comment` (string): Comments about the exception item.
+
+|No, defaults to empty array.
+
+|`description` |String |Describes the exception item. |Yes
+|`entries` |<<entries-object-schema, entries[]>> |Array containing the
+exception queries. Boolean `AND` logic is used to evaluate the relationship
+between array elements. If you want to use `OR` logic, create a separate
+exception item. |Yes
+|`list_id` |String |ID of the associated <<exceptions-api-create-container, exception container>>. |Yes
+|`item_id` |String |Unique identifier of the exception item. |No, automatically
+created when it is not provided.
+|`meta` |Object |Placeholder for metadata about the exception item. |No
+|`name` |String |The exception item's name. |Yes.
+|`namespace_type` |String a|Determines whether the exception item is available
+in all {kib} spaces or just the space in which it is created, where:
+
+* `single`: Only available in the {kib} space in which it is created.
+* `agnostic`: Available in all {kib} spaces.
+
+|No, defaults to `single`.
+|`tags` |String[] |String array containing words and phrases to help categorize
+exception items. |No 
+|`type` |String a|Exception query type, must be `simple`. |Yes
+|`_tags` |String[] a|For endpoint rules only, defines the OS on which the
+exception is implemented. Valid values are:
+
+* `os:windows`: Windows OS
+* `os:linux`: Linux OS
+* `os:macos`: Mac OS
+
+The array must also include an `endpoint` element (to implement the exception on Linux hosts, use: `["endpoint", "os:linux"]`).
+
+|For endpoint exceptions, yes. For detection exceptions, no.
+
+|==============================================
+
+[[entries-object-schema]]
+===== `entries` schema
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`field` |String |The source event field used to define the exception. |Yes
+|`list` |list |Object containing the
+<<lists-api-create-container, list container's>> `id` and `type`. Only valid for
+detection exception items.|No, except when using a list to define detection
+exceptions.
+|`operator` |String a|The operator used to determine when the exception is used.
+Can be:
+
+* `included`: The `field` has the specified value or values.
+* `excluded`: The `field` does not have specified value or values.
+
+|Yes
+
+|`type` |String a|The `type` of query:
+
+* `match`: Must be an exact match of the defined value.
+* `match_any`: Matches any of the defined values.
+* `exists`: The `field` exists.
+* `list`: The field matches values in a list container.
+* `nested`: Array of `entries` objects. Nested conditions are required for
+excluding some Endpoint fields (<<nested-field-ex-api, see example below>>).
+BEN xref lists all Endpoint fields that require the `nested` type. 
+
+|Yes
+
+|`value` |String[] |Array of field values. |Yes, except when `type` is `exists`.
```

Comment:
For `match`, `value` is string.

For `match_any`, `value` is string[]

For `type: list`, `value` is not present, instead of `value` it's `list: { id:'', type:''}`